pkg/sensors/load.go

```diff
@@ -203,6 +203,10 @@ func (s *Sensor) loadMaps(stopCtx context.Context, mapDir string) error {
 			return fmt.Errorf("map '%s' not found from '%s'", m.Name, m.Prog.Name)
 		}
 
+		if max, ok := m.Prog.MaxEntriesMap[mapSpec.Name]; ok {
```

Comment:
Maybe add "The max entries values needs to be set when map is created and then when program is loaded, so ebpf loader does not complain." ?  from your commit log as code comment too if it is not obvious... in case we try to do same or another developer, would help to not fight cryptic bpf errors... up to you

Reply:
ok, will add some more comments, thanks